omero/import_scripts/Populate_Metadata.py

```diff
@@ -26,25 +26,45 @@
 from omero.gateway import BlitzGateway
 from omero.rtypes import rstring, rlong
 import omero.scripts as scripts
-from omero.model import PlateI, ScreenI
+import omero.model
 
 import sys
 
 from omero.util.populate_roi import DownloadingOriginalFileProvider
-from omero.util.populate_metadata import ParsingContext
+try:
+    # Hopefully this will import
+    # https://github.com/ome/omero-metadata/blob/v0.3.1/src/populate_metadata.py
+    from populate_metadata import ParsingContext
```

Comment:
If this is renamed, how would you suggest doing the detection?

Reply:
Since this is a new feature I don't think we should support multiple breaking versions. If there's an agreement on renaming the module could we do that now and require that as the minimum version for these new features?